src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -221,14 +221,14 @@ public async Task Invoke(MigrateRepoCommandArgs args)
 
             var (migrationState, _, failureReason) = await githubApi.GetMigration(migrationId);
 
-            while (migrationState.Trim().ToUpper() is "IN_PROGRESS" or "QUEUED")
```

Comment:
We should do this in ado2gh MigrateRepoCommand also. And can we explicitly check for IsSucceeded (and throw a non-OctoshiftCliException if it's an unexpected state).